tests/vpp.cpp

```diff
@@ -125,27 +135,53 @@ class VppTest
         return bool(m_allocator);
     }
 
+    void flushGPU()
+    {
+        std::deque<SharedPtr<VideoFrame> > frames;
+        SharedPtr<VideoFrame> f;
+        do {
+            f = m_allocator->alloc();
+            if (!f)
```

Comment:
yes, agree with dongping, The logical here is get and sync all surface in allocator. This will make sure every gpu command finished. The allocate function in allocator only return null when no surface in the pool